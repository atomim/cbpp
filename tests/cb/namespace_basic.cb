x# = 25.0
y# = 35.0
dim sx as float
dim sy as float

namespace project <x,y,z,sx,sy>
	mult# = 100.0
	
	sx = (mult * x)/z
	sy = (mult * y)/z
endnamespace

dot sx,sy