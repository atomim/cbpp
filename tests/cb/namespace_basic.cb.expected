x# = 25.0
y# = 35.0
dim sx as float
dim sy as float

// namespace project <x,y,z,sx,sy>
	_project_mult# = 100.0
	
	sx = (_project_mult * x)/z
	sy = (_project_mult * y)/z
// endnamespace

dot sx,sy